indexer/processor.js

```diff
@@ -1227,7 +1181,7 @@ class NinaProcessor {
 
   async processCollectors() {
     try {
-      const releases = (await this.program.account.release.all()).filter(x => !blacklist.includes(x.publicKey.toBase58()));
+      const releases = await this.program.account.release.all();
```

Comment:
we will need to make sure that these releases aren't in the restricted table

Reply:
updated with check against restricted table.